### PR TITLE
purple mode

### PR DESCRIPTION
**Pull request by sarahframe**

### CODE REVIEW

**Comment by liamjohnston** [comment]
Misc comparisons:
Screenshot 2024-07-04 at 11 10 15 AM
Screenshot 2024-07-04 at 11 10 25 AM
<hr>
Screenshot 2024-07-04 at 11 10 05 AM
Screenshot 2024-07-04 at 11 09 48 AM
<hr>
Screenshot 2024-07-04 at 11 10 37 AM
Screenshot 2024-07-04 at 11 10 48 AM
<hr>
Screenshot 2024-07-04 at 11 11 12 AM
Screenshot 2024-07-04 at 11 11 22 AM

**Comment by liamjohnston** [comment]
@sarahframe Not a _huge_ deal, but is it possible to use a different icon? The checkbox icon make it look like the option is selected when it isn't 😅 
Screenshot 2024-07-04 at 11 12 49 AM

**Comment by sarahframe** [comment]
> @sarahframe Not a _huge_ deal, but is it possible to use a different icon? The checkbox icon make it look like the option is selected when it isn't 😅 Screenshot 2024-07-04 at 11 12 49 AM

That does make it look confusing. It's not possible to change the icon on a model field. We could override the wagtail template but that is more work so will add it to the bottom of my list for now

**Comment by cameralibre** [comment]
💯 I have been so confused by this exact checkbox issue on other wagtail sites recently.

**Review by cameralibre — APPROVED** [review]
💜 FED looking lovely in lilac

File: cdhweb/pages/models.py [hunk 1]
```diff
@@ -598,3 +599,15 @@ def clean_fields(self, exclude=None):
...
+
+
+@register_setting(icon="edit")
+class PurpleMode(BaseSiteSetting):
+    purple_mode = models.BooleanField(
+        default=False,
+        help_text="""This will turn the site purple 
+                                      by transforming all shades of cyan on the site into shades of purple""",
+    )
+
```
Comment (haydngreatnews) [inline cdhweb/pages/models.py:611]: Woo, tracked down a simpler icon fix (though marginally less easy to apply universally)
```suggestion

    panels = [FieldPanel("purple_mode", icon="placeholder")]
    
```

Comment (liamjohnston) [reply cdhweb/pages/models.py:611]: @sarahframe I'll leave this with you :D

**Comment by cameralibre** [comment]
I reckon that header gradient would look real nice with a bit of noise @liamjohnston...
https://css-tricks.com/grainy-gradients/

File: cdhweb/pages/models.py (hunk 1 above)
Comment (sarahframe) [reply cdhweb/pages/models.py:611]: yay thanks Haydn :D

Comment (sarahframe) [reply cdhweb/pages/models.py:611]: image

**Pull request merged**